Code/Tools/AssetProcessor/native/ui/MainWindow.ui

```diff
@@ -337,7 +337,61 @@
         <property name="currentIndex">
          <number>0</number>
         </property>
-        <widget class="QSplitter" name="jobDialogSplitter">
+         <widget class="QWidget" name="WelcomePage">
+           <layout class="QVBoxLayout" name="WelcomeVerticalLayout">
+             <item>
+              <widget class="QLabel" name="welcomeScreenTitle">
+               <property name="font">
+                <font>
+                 <pointsize>10</pointsize>
+                 <weight>75</weight>
+                 <bold>true</bold>
+                </font>
+               </property>
+               <property name="text">
+                <string># Welcome to the O3DE Asset Processor</string>
+               </property>
+               <property name="textFormat">
+                <enum>Qt::MarkdownText</enum>
+               </property>
+              </widget>
+             </item>
+             <item>
+               <widget class="QLabel" name="WelcomeDescriptionLabel">
+                 <property name="text">
+                   <string>This tool converts an O3DE project's source assets to game ready content.</string>
+                 </property>
+               </widget>
+             </item>
+             <item>
+               <widget class="QLabel" name="WelcomeDescriptionLabel">
```

Comment:
I've duplicated this widget name, adding this comment so I don't forget to fix it